build.gradle

```diff
@@ -3,6 +3,8 @@ plugins {
     id 'org.ajoberstar.reckon' version "0.13.0"
 }
 
+version = "2.0.0"
```

Comment:
:question: You can't set a version and use reckon at the same time.

Reply:
committed per accident :upside_down_face: